addons/website/static/src/builder/plugins/translation_plugin.js

```diff
@@ -56,13 +56,18 @@ export class TranslationPlugin extends Plugin {
         clean_for_save_handlers: this.cleanForSave.bind(this),
         get_dirty_els: this.getDirtyTranslations.bind(this),
         after_setup_editor_handlers: () => {
-            const translationSavableEls = getTranslationEditableEls(
-                this.services.website.pageDocument
-            );
-            for (const translationSavableEl of translationSavableEls) {
-                if (!translationSavableEl.hasAttribute("data-oe-readonly")) {
-                    translationSavableEl.classList.add("o_editable");
+            if (this.config.isTranslation) {
```

Comment:
Hello @divy-odoo :wave: 
Just passing by and saw that you are using `this.config.isTranslation`. Just to warn you that it has been removed from the config by [this commit](https://github.com/odoo/odoo/commit/281ce591d4271a15dc867b439365cb41e9dee0b4). However, this condition is now always true as the `TranslationPlugin` plugin is only instantiated in translate mode.